error when deploy

```
  src/bot/commands/like.ts(4,2): error TS6133: 'CallbackQuery' is declared but its value is never read.
       src/bot/commands/like.ts(16,64): error TS2702: 'CallbackQuery' only refers to a type, but is being used as a namespace here.
       src/bot/commands/settings/buttons_handler.ts(4,10): error TS6133: 'CallbackQuery' is declared but its value is never read.
       src/bot/commands/settings/buttons_handler.ts(26,85): error TS2702: 'CallbackQuery' only refers to a type, but is being used as a namespace here.
       src/bot/commands/fix_instant_view.ts(18,10): error TS6133: 'CallbackQuery' is declared but its value is never read.
       src/bot/commands/fix_instant_view.ts(22,18): error TS2702: 'CallbackQuery' only refers to a type, but is being used as a namespace here.
       src/bot/commands/fix_instant_view.ts(33,18): error TS2702: 'CallbackQuery' only refers to a type, but is being used as a namespace here.
       src/bot/commands/fix_instant_view.ts(235,18): error TS2702: 'CallbackQuery' only refers to a type, but is being used as a namespace here.
       src/bot/callback_handler.ts(4,18): error TS6133: 'CallbackQuery' is declared but its value is never read.
       src/bot/callback_handler.ts(13,85): error TS2702: 'CallbackQuery' only refers to a type, but is being used as a namespace here.
-----> Build failed

```

heroku deploy.

--- a/src/bot/callback_handler.ts
+++ b/src/bot/callback_handler.ts
@@ -10,8 +10,8 @@ import help, { searchTips } from './commands/help'
 import settingsChanger      from './commands/settings/buttons_handler'
 import fixInstantViewAsync  from './commands/fix_instant_view'
 
-export default async function callbackHandler(ctx: Context<Update>, callback_query: CallbackQuery.DataCallbackQuery): Promise<void> {
-	const data: string = callback_query.data
+export default async function callbackHandler(ctx: Context<Update>, callback_query: CallbackQuery): Promise<void> {
+	const data: string = callback_query.data || ''
 	if (data === 'r'){
 		try {
 			await makeRandom(ctx,'next')

--- a/src/bot/commands/fix_instant_view.ts
+++ b/src/bot/commands/fix_instant_view.ts
@@ -19,7 +19,7 @@ import { CallbackQuery, InlineKeyboardButton } from 'telegraf/typings/core/types
 
 export default async function fixInstantViewAsync(
 	ctx: Context,
-	callback_query: CallbackQuery.DataCallbackQuery
+	callback_query: CallbackQuery
 ): Promise<void> {
 	try {
 		await fixInstantView(ctx, callback_query)
@@ -30,8 +30,10 @@ export default async function fixInstantViewAsync(
 
 async function fixInstantView(
 	ctx: Context,
-	callback_query: CallbackQuery.DataCallbackQuery
+	callback_query: CallbackQuery
 ): Promise<void> {
+	callback_query.data = callback_query.data || ''
+
 	// Get doujin's id
 	const matchId = callback_query.data.match(/_([0-9]+)/)
 	if (!matchId || !Number(matchId[1])) {
@@ -232,7 +234,7 @@ async function getPages(id: number | string): Promise<string[]> {
 async function buildKeyboardBack(
 	telegraph_url: string | undefined,
 	id: string,
-	callback_query: CallbackQuery.DataCallbackQuery,
+	callback_query: CallbackQuery,
 	message: Message | undefined | null
 ): Promise<InlineKeyboardButton[][]> {
 	const fixing_keyboard: InlineKeyboardButton[][] = [[]]

--- a/src/bot/commands/like.ts
+++ b/src/bot/commands/like.ts
@@ -13,13 +13,15 @@ import i18n   from '../../lib/i18n'
 import saveAndGetUser  from '../../db/save_and_get_user'
 import saveAndGetManga from '../../db/save_and_get_manga'
 
-export default async function likeDoujin (ctx: Context, query: CallbackQuery.DataCallbackQuery): Promise<void> {
+export default async function likeDoujin (ctx: Context, query: CallbackQuery): Promise<void> {
 	let user: User | undefined
 	try {
 		user = await saveAndGetUser(ctx)
 	} catch (error) {
 		throw new Werror(error, 'Getting user in callbackHandler')
 	}
+
+	query.data = query.data || ''
 
 	const doujinId = Number(query.data.split('_')[1])
 	if(!doujinId){

--- a/src/bot/commands/settings/buttons_handler.ts
+++ b/src/bot/commands/settings/buttons_handler.ts
@@ -23,14 +23,14 @@ async function safeAndEdit(user: User, ctx: Context) {
 		throw new Werror(error, 'Editing message after changing settings')
 	}
 }
-export default async function settingsChanger(ctx: Context<Update>, callback_query: CallbackQuery.DataCallbackQuery): Promise<void> {
+export default async function settingsChanger(ctx: Context<Update>, callback_query: CallbackQuery): Promise<void> {
 	let user: User | undefined
 	try {
 		user = await saveAndGetUser(ctx)
 	} catch (error) {
 		throw new Werror(error, 'Getting user in settings')
 	}
-	const data = callback_query.data
+	const data = callback_query.data || ''
 	const setting = data.split('_')[1]
 	switch (setting) {
 	case 'search-type': {

